nixos/tests/matrix/mautrix-meta-postgres.nix

```diff
@@ -92,6 +92,7 @@ import ../make-test-python.nix ({ pkgs, ... }:
             };
 
             bridge.permissions."@${userName}:server" = "user";
+            bridgemanagement_room_text.welcome = "Hello, I'm an mautrix bridge bot.";
```

Comment:
There is no `bridgemanagement_room_text.welcome` option in mautrix-meta. Closest is `bridge.management_room_text.welcome`. Please run the tests after making changes to either the module or the changes. This, along the different checked message, fails the tests.

This message was to signify that this bot goes with the instagram instance `services.mautrix-meta.instances.instagram`, and not any other. If we were to add tests for different type than instagram, it could then have that welcome message instead, so it's obvious what bot is talking. I do not see a reason for this change.